<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChangeAttributesDialog</class>
 <widget class="QDialog" name="ChangeAttributesDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>320</width>
    <height>240</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QCheckBox" name="checkBox_inludeTime">
     <property name="toolTip">
      <string>Attributes will include date and time for each file in archive. This affects only files added later.</string>
     </property>
     <property name="text">
      <string>Include file time/date</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="checkBox_includeCRC32">
     <property name="toolTip">
      <string>Attributes will contain CRC32 for each file in archive.</string>
     </property>
     <property name="text">
      <string>Include file CRC32</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="checkBox_includeMD5">
     <property name="toolTip">
      <string>Attributes will contain MD5 for each file in archive.</string>
     </property>
     <property name="text">
      <string>Include file MD5</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="checkBox_updateChecksums">
     <property name="toolTip">
      <string>After changing Attributes, CRC32 and MD5 will be updated for each file in archive.</string>
     </property>
     <property name="text">
      <string>Update checksums for each file in archive</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ChangeAttributesDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ChangeAttributesDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
